source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
> My assumption is following: if the developer wants to use Likely as a Commonjs module he's responsible for manual init, seems it's true in many cases. If the developer doesn't want to think about modules and makes a simple website, he just drops the script to the page and Likely works by itself (like now). 

Yep, and this is how it works now. The problem with that IIFE approach is that the event listener is set even if Likely is used as a CommonJS module.

> I'm not against two bundles, btw :)

Yup, I see this :–)